src/styles/tabs.js

```diff
@@ -1,12 +1,17 @@
 /* @flow strict-local */
 import { BRAND_COLOR } from './constants';
 
+/**
+ * react-navigation tabNavigation also treat styles as {}
+ * so match the same type here too
+ */
```

Comment:
Nit: Let's reserve the `/**` jsdoc marker for comments that are at the level of clarity and information that makes them a proper doc comment. This comment isn't and that's 100% fine -- this type doesn't need a doc comment -- but we'll just write it with plain `/*` or `//` to keep that clear.

See for comparison [this style-guide item](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use--for-public-quality-private-documentation) from Flutter. The context is a little different, but this point applies just as well for us.

Reply:
Separately: let's put this comment immediately above the `style` property itself.